recipe/meta.yaml

```diff
@@ -1,176 +1,131 @@
+      - lib/cmake/openblas/*                              # [not win]
+      - Library/include/openblas                          # [win]
+      - Library/lib/openblas{{ SYMBOLSUFFIX }}.lib        # [win]
+      - Library/lib/pkgconfig/openblas.pc                 # [win]
+      - Library/share/cmake/OpenBLAS                      # [win]
     requirements:
       run:
-        - {{ pin_subpackage("libopenblas", exact=True) }}
-        - {{ pin_subpackage("nomkl", exact=True) }}
-    about:
-      home: https://www.openblas.net/
-      license: BSD-3-Clause
-      license_file: LICENSE
-      license_family: BSD
-      summary: OpenBLAS headers and libraries for developing software that used OpenBLAS.
-      description: OpenBLAS is based on GotoBLAS2 1.13 BSD version.
-      doc_url: https://www.openblas.net/
-      doc_source_url: https://github.com/xianyi/OpenBLAS/blob/develop/README.md
-      dev_url: https://github.com/xianyi/OpenBLAS
-
-  # openblas meta-package. It is better to require openblas-devel or libopenblas
-  - name: openblas
-    requirements:
-      run:
-        - {{ pin_subpackage("libopenblas", exact=True) }}
-        - {{ pin_subpackage("openblas-devel", exact=True) }}
-        - blas=*=openblas
-        - {{ pin_subpackage("nomkl", exact=True) }}
-
-    about:
-      home: https://www.openblas.net/
-      license: BSD-3-Clause
-      license_file:
-        - LICENSE
-        - lapack-netlib/LICENSE
-      license_family: BSD
-      summary: An optimized BLAS library
-      description: OpenBLAS is based on GotoBLAS2 1.13 BSD version.
-      doc_url: https://www.openblas.net/
-      doc_source_url: https://github.com/xianyi/OpenBLAS/blob/develop/README.md
-      dev_url: https://github.com/xianyi/OpenBLAS
-
-  # mutex package to keep only one blas implementation in a given env
-  # Since this package has a non-unique name, blas-1.0-openblas.conda, it is omitted from future updates as they cause issues
-  # with overwrites on the PBP. If an update to this package is needed in the future, the name should have a hash added
-  # so that it becomes unique per publish.
-#  - name: blas
-#    version: 1.0
-#    build:
-#      string: openblas
-#      # track_features doesn't really track anything anymore (blas metapackage
-#      # dependencies do the same job better). This is still here, though, as it
-#      # effectively "weighs down" nomkl packages, allowing mkl to take
-#      # precedence when defaults is the top channel priority.
-#      track_features:
-#        - nomkl
+        - {{ pin_subpackage("libopenblas" ~ name_suffix, exact=True) }}
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libopenblas{{ SYMBOLSUFFIX }}.a                       # [not win]
+        - test -f ${PREFIX}/lib/libopenblas{{ SYMBOLSUFFIX }}{{ SHLIB_EXT }}          # [not win]
+        - if not exist %PREFIX%\\Library\\lib\\openblas{{ SYMBOLSUFFIX }}.lib exit 1  # [win]
 
-  - name: nomkl
```

Comment:
Pytorch is a downstream package that relies on `nomkl`:
```
Downstream build order (groups:() feedstocks:() packages:('nomkl',) allow_list:() block_list:() drop_noarch:False sections:('build', 'host', 'run')):
000 openblas-feedstock             ['openblas-devel', 'openblas']
001 pytorch-feedstock              ['pytorch']
```
see also https://github.com/AnacondaRecipes/pytorch-feedstock/blob/master/recipe/meta.yaml#L385